### ISSUE TITLE
tests.em.protocols.test_protocol_multireference_alignability.TestMultireferenceAlignability

### ISSUE THREAD

**Issue by pconesa** [I2PC/scipion#1284]
Fail on buildbot
facilities branch

**Comment by 1aviervargas** [comment]
I am going to take a look, thanks pablo

**Comment by 1aviervargas** [comment]
I have taken a look to the issue and I do not get any error from this test. I am using branch  release-1.1.facilities-devel

![screenshot_20171030_173944](https://user-images.githubusercontent.com/12271201/32183468-a9bc3056-bd99-11e7-9e0d-35932b304bec.png)

**Comment by pconesa** [comment]
Hi, @1aviervargas thanks for having a look...I'll check what is wrong with it in buildbot, and come back.

**Comment by yaizar** [comment]
Looks like this is passing in buildbot too as of now?

![image](https://user-images.githubusercontent.com/4609687/32947251-4a2a6f6c-cb9b-11e7-9f62-711fb309417e.png)

**Comment by azazellochg** [comment]
> 60[   FAILED ] TestMultireferenceAlignability.test_validate
> 61
> 62Traceback (most recent call last):
> 63  File "/home/buildbot-slave/data/slave/release11-facilities_Protocol_Tests/build/software/lib/python2.7/unittest/case.py", line 329, in run
> 64    testMethod()
> 65  File "/home/buildbot-slave/data/slave/release11-facilities_Protocol_Tests/build/pyworkflow/tests/em/protocols/test_protocol_multireference_alignability.py", line 118, in test_validate
> 66    self.assertTrue(firstVol.weightAlignabilityAccuracy > 0.2)
> 67  File "/home/buildbot-slave/data/slave/release11-facilities_Protocol_Tests/build/software/lib/python2.7/unittest/case.py", line 422, in assertTrue
> 68    raise self.failureException(msg)
> 69AssertionError: False is not true

**Comment by yaizar** [comment]
Fails when importing particles from relion:
```
00001:   RUNNING PROTOCOL -----------------
00002:             PID: 50605
00003:         Scipion: v1.2.beta (2018-02-22) Caligula (release-1.1.facilities-fixes 701681b)
00004:      currentDir: /home/scipion/ScipionUserData/projects/TestMultireferenceAlignability
00005:      workingDir: Runs/000162_ProtImportParticles
00006:         runMode: Continue
00007:             MPI: 1
00008:         threads: 1
00009:   referenceFile: /home/local/scipion/data/tests/relion_tutorial/import/case2/relion_it015_data.star
00010:   checking:  /home/local/scipion/data/tests/relion_tutorial/import/case2/particles.mrcs
00011:    >>>> found!!!
00012:   len(steps) 1 len(prevSteps) 0 
00013:    Starting at step: 1
00014:    Running steps 
00015:   STARTED: importParticlesStep, step 1
00016:     2018-03-27 15:24:17.342859
00017:   referenceFile: /home/local/scipion/data/tests/relion_tutorial/import/case2/relion_it015_data.star
00018:   checking:  /home/local/scipion/data/tests/relion_tutorial/import/case2/particles.mrcs
00019:    >>>> found!!!
00020:   Traceback (most recent call last):
00021:     File "/home/local/scipion/pyworkflow/protocol/protocol.py", line 186, in run
00022:       self._run()
00023:     File "/home/local/scipion/pyworkflow/protocol/protocol.py", line 233, in _run
00024:       resultFiles = self._runFunc()
00025:     File "/home/local/scipion/pyworkflow/protocol/protocol.py", line 229, in _runFunc
00026:       return self._func(*self._args)
00027:     File "/home/local/scipion/pyworkflow/em/protocol/protocol_import/particles.py", line 185, in importParticlesStep
00028:       ci.importParticles()
00029:     File "/home/local/scipion/pyworkflow/em/packages/relion/dataimport.py", line 79, in importParticles
00030:       readAcquisition=False, alignType=self.alignType)
00031:     File "/home/local/scipion/pyworkflow/em/packages/relion/convert.py", line 584, in readSetOfParticles
00032:       partSet.append(img)
00033:     File "/home/local/scipion/pyworkflow/em/data.py", line 889, in append
00034:       EMSet.append(self, image)
00035:     File "/home/local/scipion/pyworkflow/object.py", line 1150, in append
00036:       self._insertItem(item)
00037:     File "/home/local/scipion/pyworkflow/object.py", line 1154, in _insertItem
00038:       self._getMapper().insert(item)
00039:     File "/home/local/scipion/pyworkflow/mapper/sqlite.py", line 729, in insert
00040:       *obj.getObjDict().values())
00041:     File "/home/local/scipion/pyworkflow/mapper/sqlite.py", line 1119, in insertObject
00042:       self.executeCommand(self.INSERT_OBJECT, args)
00043:   ProgrammingError: Incorrect number of bindings supplied. The current statement uses 37, and there are 25 supplied.
00044:   Protocol failed: Incorrect number of bindings supplied. The current statement uses 37, and there are 25 supplied.
00045:   FAILED: importParticlesStep, step 1
00046:     2018-03-27 15:24:18.548405
00047:   ------------------- PROTOCOL FAILED (DONE 1/1)
```
I'll look further into it..

**Comment by azazellochg** [comment]
I guess it works now :)
